docs/src/main/sphinx/installation/try-it.rst

```diff
@@ -0,0 +1,265 @@
+Try Trino with Docker
+======================
+
+This guide focuses on your first ten minutes
+using Trino, and relies on Docker to simplify the installation and
+ensure that the CLI runs on your platform of choice.
+
+This Docker container includes both the Trino server and the 
+Trino command line interface (CLI), which provides a terminal-based,
+interactive shell for running queries and inspecting catalog structures
+in large data clusters. In production, it's unlikely that your server 
+and client will reside in the same location. However, to keep this guide 
+short and sweet, we are going to run both from your machine.
+
+In this guide, you'll learn how to:
+
+-  Install and run the Trino Interactive CLI
+-  Call Trino from the terminal
```

Comment:
"Call" evokes the programming concept. We just:

"Start Trino from the terminal"

Reply:
Also, reverse the order of the two bullets, because that's the order encountered below. The CLI is one of many ways to access the Trino cluster, which is the important thing, and must be started first.